libs/ballot-interpreter/src/interpret_nh_hmpb.test.ts

```diff
@@ -222,16 +234,20 @@ describe('HMPB - m17 backup', () => {
             definite: 0.07,
             writeInTextArea: 0.05,
           },
-          adjudicationReasons: [AdjudicationReason.Overvote],
+          adjudicationReasons: [
+            AdjudicationReason.Overvote,
+            AdjudicationReason.UnmarkedWriteIn,
+          ],
         },
         sheet
       );
 
       const interpretation = mapSheet(interpretationWithImages, (page) => {
         // Only snapshot stable fields - marks and layout coordinates may change slightly on CI
         if (page.interpretation.type === 'InterpretedHmpbPage') {
-          const { type, metadata, votes } = page.interpretation;
-          return { type, metadata, votes };
+          const { type, metadata, votes, unmarkedWriteIns } =
+            page.interpretation;
+          return { type, metadata, votes, unmarkedWriteIns };
```

Comment:
Arguably problematic that we're including unmarkedWriteIns unedited in the snapshot because it includes scores, but the scores are rounded to two decimal places so I think they should be consistent on CI.